wicket-user-guide/src/main/asciidoc/whyLearn/whyLearn_4.adoc

```diff
@@ -3,7 +3,7 @@ Wicket is not the only component oriented framework available in the Java ecosys
 
 * *Wicket is 100% open source*: Wicket is a top Apache project and it doesn't depend on any private company. You don't have to worry about future licensing changes, Wicket will always be released under Apache license 2.0 and freely available.
 
-* *Wicket is a community driven project*: The Wicket team supports and promotes the dialogue with the framework's users through two mailing lists  http://wicket.apache.org/help/email.html[(one for users and another one for framework developers)] and an  https://issues.apache.org/jira/browse/WICKET[Apache JIRA] (the issue tracking system). Moreover, as any other Apache project, Wicket is developed paying great attention to user feedbacks and to suggested features.
+* *Wicket is a community driven project*: The Wicket team supports and promotes the dialogue with the framework's users through two mailing lists  http://wicket.apache.org/help/email.html[(one for users and another one for framework developers)] and an  https://issues.apache.org/jira/browse/WICKET[Apache JIRA] (the issue tracking system). Moreover, as any other Apache project, Wicket is developed paying great attention to user feedback and to suggested features.
```

Comment:
>    "Feedback" is a mass (or non-count, or uncountable) noun. As such, it has no plural form.

https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&cad=rja&uact=8&ved=2ahUKEwiGnOqahOreAhXsKcAKHdHTDBcQFjABegQIBhAE&url=https%3A%2F%2Fwww.quora.com%2FWhat-is-the-plural-of-feedback-Can-you-ever-use-feedbacks&usg=AOvVaw1HSV_kZmkIDSmShhBMaoDN